praw/__init__.py

```diff
@@ -102,6 +102,7 @@ class Config(object):  # pylint: disable=R0903
                  'flairselector':       'api/flairselector/',
                  'flairtemplate':       'api/flairtemplate/',
                  'friend':              'api/friend/',
+                 'friend_v1':           'api/v1/me/friends/%s',
```

Comment:
@bboe Unless I'm missing something simple, wouldn't using the format syntax cause it to break for python 2?